website/docs/cloud-docs/api-docs/policy-sets.mdx

```diff
@@ -941,6 +1020,54 @@ curl \
   https://app.terraform.io/api/v2/policy-sets/polset-3yVQZvHzf5j3WRJ1/relationships/workspaces
 ```
 
+## Detach the Policy Set from exclusions
+
+`DELETE /policy-sets/:id/relationships/excludedWorkspaces`
+
+| Parameter | Description                                                                                                   |
+| --------- | ------------------------------------------------------------------------------------------------------------- |
+| `:id`     | The ID of the policy set to detach from excluded workspaces. Use the "List Policy Sets" endpoint to find IDs. |
```

Comment:
The negative phrasing is a bit confusing here. This effectively just returns a policy set's enforcement back to normal state, right? Can you please add something to that effect?

Reply:
@rkoron007 would this be better: The ID of the policy set reattach to excluded workspaces.

Reply:
I think the verb "reattach" feels confusing here. It's excluded and reincluded, right?

```suggestion
| `:id`     | The ID of the policy set Terraform Cloud should reinclude (enforce) on the specified workspaces. Use the "List Policy Sets" endpoint to find IDs. |
```